JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -136,16 +145,27 @@ ReliabilityAnalysis <- function(dataset = NULL, options, perform = "run",
 
 		# calculate chronbach alpha and gutmanns lambda6
 		r <- psych::alpha(d, key = key)
+		
+		# calculate the greatest lower bound -- only possible for more than 2 variables.
+		if (length(variables) < 3) {
+			
+			r[["glb"]] <- "."
+			
+		} else { # try since the glb is rather error prone...
+			
+			r[["glb"]] <- try(psych::glb(r = d, key = key)[["glb.max"]], silent = TRUE)
+			
+		}
 
 		# calculate McDonalds omega
-		omega <- psych::omega(d, 1, flip = FALSE)[["omega.tot"]]
+		omega <- psych::omega(d, 1, flip = FALSE, plot = FALSE)[["omega.tot"]]
```

Comment:
Just a suggestion: it might be good to always wrap calls to external packages in a try. But you know the behavior of these packages and the intended behaviour when an error occurs in your output table better than me.